### DISCUSSION TITLE
Training on multiple time-series and predicting on one

### DISCUSSION THREAD

**Discussion by miteshyadav** [awslabs/gluonts#1507] [Q&A]
Is there a way to pass a single time series data to predict in the test dataset, instead of passing values for all of the time series at once, if I need to make a prediction on just one dataset?

Can I just pass None to the rest of the fields that are not needed and get it to work? Im trying to figure out a way to speed up the predictions and not overload the predictor with test data for time series that is not needed

**Comment by xrl0123** [comment]
Anyone knows the answer to this question? I tried predicting just one item (make_evaluation_predictions) using a DeepAR model trained on multiple items. I got different forecast result this way compared to doing the prediction on all the items at once. I am trying to figure out if it is doable to split the data into batches and have them processed in paralell?
